modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -93,75 +101,141 @@ definition {
 	@description = "This is a test for LPS-175292. Verify that is possible to export all visible Claims"
 	@priority = 4
 	test CanExportAllVisibleClaims {
-		task ("Given I go to add a new MDF Request with two activities") {
-			PRMUtils.gotoPRM();
+		task ("Given that the Manager user is logged in and goes to MDF listing page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
 
-			PRMNavigator.goToNavItem(navItem = "Marketing");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
 
-			PRMMarketing.gotoAddNewMDF();
+		task ("And the manager approves the MDF") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
 
-			PRMMDFRequest.fillGoalsPage(
-				accountName = "Company Name 1",
-				campaignDescription = "Campaign Description",
-				campaignName = "Campaign Name 2",
-				countryName = "Global",
-				goalsOptions = "Lead generation",
-				targetMarketOptions = "Education,Energy",
-				targetRolesOptions = "Administrator");
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
 
-			Button.click(button = "Continue");
+		task ("And Login with the Partner user and go to Partner Portal") {
```

Comment:
Hi @marinaasant0s, it's a little confuse. I think it should be `'And the partner logs in...'`